src/command/info/info.c

```diff
@@ -444,16 +457,28 @@ stanzaInfoList(const String *stanza, StringList *stanzaList, const String *backu
         kvPut(varKv(stanzaInfo), STANZA_KEY_BACKUP_VAR, varNewVarLst(backupSection));
         kvPut(varKv(stanzaInfo), KEY_ARCHIVE_VAR, varNewVarLst(archiveSection));
 
+        // If a status has not already been set, check if there's a local backup running
+        static bool runningBackup = false;
+        if (kvGet(varKv(stanzaInfo), STANZA_KEY_STATUS_VAR) == NULL)
+        {
+            // Try to acquire a lock. If not possible, assume another backup is already running.
+            static LockType lockTypeToFind = lockTypeBackup;
```

Comment:
This declaration isn't needed.  Just put `lockTypeBackup` in the function below.

Reply:
I think I already tested that and it didn't work. I've just retested it now (post-rebase) and it works indeed. Modified. Thanks